rotkehlchen/chain/polygon_pos/tokens.py

```diff
@@ -0,0 +1,19 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.tokens import EvmTokens
+from rotkehlchen.types import ChecksumEvmAddress
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+    from .node_inquirer import PolygonPOSInquirer
+
+
+class PolygonPOSTokens(EvmTokens):
+
+    def __init__(self, database: 'DBHandler', polygon_pos_inquirer: 'PolygonPOSInquirer') -> None:
+        super().__init__(database=database, evm_inquirer=polygon_pos_inquirer)
+
+    # -- methods that need to be implemented per chain
+    def _get_token_exceptions(self) -> list[ChecksumEvmAddress]:
```

Comment:
Hmmmm again here you don't do anything and you don't even leave a TODO.

This we will just forget it even needed to do something.

```python
    def _get_token_exceptions(self) -> list[ChecksumEvmAddress]:
        exceptions = [A_OPTIMISM_ETH.resolve_to_evm_token().evm_address]
        with self.db.conn.read_ctx() as cursor:
            ignored_asset_ids = self.db.get_ignored_asset_ids(cursor=cursor)

        # TODO: Shouldn't this query be filtered in the DB?
        for asset_id in ignored_asset_ids:  # don't query for the ignored tokens
            if (evm_details := asset_id_is_evm_token(asset_id)) is not None and evm_details[0] == ChainID.OPTIMISM:  # noqa: E501
                exceptions.append(evm_details[1])

        return exceptions
```

As you can see above the code for this in optimism is also simple. And it's even a sort of a copy of the mainnet ones. So I think you should even abstract it.

Reply:
Makes sense